lib/searchable_by/column.rb

```diff
@@ -1,33 +1,50 @@
 module SearchableBy
   class Column
-    attr_reader :attr, :type, :match
+    attr_reader :attr, :type, :match, :match_phrase
     attr_accessor :node
 
-    def initialize(attr, type: :string, match: :all)
+    def initialize(attr, type: :string, match: :all, match_phrase: nil)
       @attr  = attr
       @type  = type.to_sym
       @match = match
+      @match_phrase = match_phrase || match
     end
 
     def build_condition(value)
+      scope = node.not_eq(nil)
+
       case type
       when :int, :integer
-        begin
-          node.not_eq(nil).and(node.eq(Integer(value.term)))
-        rescue ArgumentError
-          nil
-        end
+        int_condition(scope, value)
+      else
+        str_condition(scope, value)
+      end
+    end
+
+    private
+
+    def int_condition(scope, value)
+      scope.and(node.eq(Integer(value.term)))
+    rescue ArgumentError
+      nil
+    end
+
+    def str_condition(scope, value)
+      term = value.term.dup
+      type = value.phrase ? match_phrase : match
```

Comment:
Hm, maybe then don't have both `match: ...` and `match_phrase: ...`?

Just: `match: :prefix/:exact`?

Reply:
They serve two different purposes, e.g. you can set `match: :prefix, match_phrase: :exact`. Then, if someone searches for `two words` it will match results with `two%` or `words%`. If someone searches a whole phrase i.e. `"two words"` then it will match results where the field is `"two words"` exactly.